R/quantity-type.R

```diff
@@ -35,9 +35,8 @@ getQuantityTypeAsString <- function(quantityInt){
   quantityTypes <- which(as.logical(quantityTypes))
   # Convert to integer to get values from the QuantityType-enum.
   # -2 because QuantityType starts at 2 and not 0
-  quantityTypes <- bitwShiftL(2, quantityTypes-2)
+  quantityTypes <- bitwShiftL(2, quantityTypes - 2)
 
   quantityTypes <- lapply(quantityTypes, getEnumKey, QuantityType)
-  #Collapse the separate entries to a string separated by a comma

```

Comment:
@PavelBal I remove this comment because it's obvious. Good job on describing the other lines however